File: test_archives/cpp_functions/batch2/function5.cpp
# C++ Code
#include <vector>
#include <string>

std::pair<double, std::vector<std::string>> function5(int int1, float float1, std::string str1, std::vector<std::string> list1) {
    double result = int1 + float1;

    std::vector<std::string> new_list;
    for (int i = 0; i < list1.size(); i++) {
        new_list.push_back(str1 + list1[i]);
    }

    return std::make_pair(result, new_list);
}

File: test_archives/py_functions/batch2/function5.py
def function5(int1, float1, str1, list1): 
    # This function takes in an integer, float, string and list as parameters
    # and returns the sum of the first two parameters
    result = int1 + float1

    # The string and list are used to construct a new list
    new_list = []
    for i in range(len(list1)):
        new_list.append(str1 + list1[i])

    # Return the sum of the first two parameters and the new list
    return result, new_list